learning/data-types.md

```diff
@@ -51,15 +57,24 @@ Something about sequences
 
 ### String {#string}
 
-Explaining strings
+Strings are just text, like what you're reading right now. You define a string by putting your text around quotation marks, for example
```

Comment:
also, one thing that confuses many is 'this should contain something that makes more sense than what I'm typing right now' is also a string in python, not a char